elisp/intero.el

```diff
@@ -2660,6 +2666,14 @@ For debugging purposes, try running the following in your terminal:
         (setq intero-ghc-version
               (intero-ghc-version-raw)))))
 
+(defun intero-ghci-output-flags ()
+  "Get the appropriate ghci output flags for the current GHC version"
+  (with-current-buffer (intero-buffer 'backend)
```

Comment:
I did just cargo cult this `with-current-buffer`. Seems like it might be needed for `intero-ghc-version`